extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/pages/azureSettingsPage.ts

```diff
@@ -293,9 +284,9 @@ export class AzureSettingsPage extends BasePage {
 	}
 
 	private async createServerDropdown(view: azdata.ModelView) {
-		this._serverGroupDropdown = view.modelBuilder.dropDown().withProperties({
-			required: true,
-		}).component();
+		this._serverGroupDropdown = view.modelBuilder.dropDown()
```

Comment:
Where is the message that will be shown to the user when the validation fails specified?